src/lib/utils/helpers/handle-clicks.helper.ts

```diff
@@ -1,94 +1,94 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import { isFolderCreatedFirstTime } from "$lib/store/collection";
-import { CollectionsViewModel } from "@app/pages/Collections/Collections.ViewModel.old";
-import { ItemType } from "../enums/item-type.enum";
-import type { Path } from "../interfaces/request.interface";
-import { generateSampleCollection } from "../sample/collection.sample";
-import { generateSampleFolder } from "../sample/folder.sample";
-import { generateSampleRequest } from "../sample/request.sample";
-import { moveNavigation } from "./navigation";
-const _collectionMethods = new CollectionsViewModel();
-export const handleCollectionClick = (
-  collection: any,
-  currentWorkspaceId: string,
-  collectionId: string,
-) => {
-  let totalFolder: number = 0;
-  let totalRequest: number = 0;
-  collection.items.map((item) => {
-    if (item.type === ItemType.REQUEST) {
-      totalRequest++;
-    } else {
-      totalFolder++;
-      totalRequest += item.items.length;
-    }
-  });
+// /* eslint-disable @typescript-eslint/no-explicit-any */
+// import { isFolderCreatedFirstTime } from "$lib/store/collection";
+// import { CollectionsViewModel } from "../../../pages/Collections/Collections.ViewModel.old";
+// import { ItemType } from "../enums/item-type.enum";
+// import type { Path } from "../interfaces/request.interface";
+// import { generateSampleCollection } from "../sample/collection.sample";
+// import { generateSampleFolder } from "../sample/folder.sample";
+// import { generateSampleRequest } from "../sample/request.sample";
+// import { moveNavigation } from "./navigation";
```

Comment:
Why this whole file is commented?

Reply:
This file is not being used in code, all necessary functionalities from this file are transferred to CollectionsPage.ViewModel.ts. This is here only for reference for now.